crates/typst-cli/src/args.rs

```diff
@@ -153,6 +153,10 @@ pub struct QueryCommand {
     /// The format to serialize in
     #[clap(long = "format", default_value = "json")]
     pub format: SerializationFormat,
+
+    /// Pretty-print JSON
```

Comment:
```suggestion
    /// Whether to pretty-print the serialized output
```
It could apply to potential future formats